vars/util.groovy

```diff
@@ -204,4 +204,22 @@ GIT INFORMATION REPORT
     } else {
         println '[WARNING] The variable GIT_INFORMATION_REPORT does not exist'
     }
+}
+
+def getNextSnapshot(String version) {
+    String [] versionSplit = version.split("\\.")
+    if(versionSplit.length == 3) {
+        if(versionSplit[0].isNumber() && versionSplit[1].isNumber() && versionSplit[2].isNumber()) { 
+            int majorVersion = Integer.parseInt(versionSplit[0])
+            int minorVersion = Integer.parseInt(versionSplit[1])
+            int microVersion = Integer.parseInt(versionSplit[2])
+            return "${majorVersion}.${minorVersion}.${microVersion+1}-SNAPSHOT"
+        }
+        else {
+            error "Version is not in the format X.Y.Z"
```

Comment:
Could you also add a test for this error ?

Reply:
This is actually the error that should be `error "Version is not in the required format.It should contain only numeric characters`. The other error is when there are more than 3 dots. Please update these error messages and the method names used to test them.